### DISCUSSION TITLE
Home Assistant/Telegram : formattazione msg con attributi delle entità

### DISCUSSION THREAD

**Discussion by micium** [indomus/forum#1651] [HUB: Home Assistant]
Buongiorno a tutti,
stavo mettendo mano ad una automazione di notifica aggiornamento pacchetti HACS, volendo aggiungere nel messaggio di notifica tramite telegram anche quale pacchetto è da aggiornare, da quale versione a quale versione.
Non riesco formattare il seguente messaggio con telegram, funzionante nei modelli dello strumento degli sviluppatori, via whatsapp e via gmail, quando lo stato del sensore "sensor.hacs > 0" :

{{state_attr('sensor.hacs','repositories')[0]['display_name'] }} dalla {{state_attr('sensor.hacs','repositories')[0]['installed_version'] }} alla {{state_attr('sensor.hacs','repositories')[0]['available_version'] }}

Qualcuno ha qualche suggerimento o link dove poter approfondire la formattazione dei messaggi con Telegram.

Grazie

**Comment by 50494554524F** [comment]
https://core.telegram.org/bots/api#html-style
o
https://core.telegram.org/bots/api#markdownv2-style

**Comment by micium** [comment]
Risolto cambiando il messaggio con template come di seguito:

    {{ states.sensor.hacs.attributes.repositories.0.display_name }} da {{
    states.sensor.hacs.attributes.repositories.0.installed_version }} a {{
    states.sensor.hacs.attributes.repositories.0.available_version }}
